### PR TITLE
Feature/ted 512

### PR DESCRIPTION
**Pull request by kaleanych**

### CODE REVIEW

File: dags/fetch_notices_for_date_range.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 
 from dags import DEFAULT_DAG_ARGUMENTS
-from airflow.decorators import dag, task
-from airflow.operators.python import get_current_context
-
 from dags.fetch_notices_per_day_worker import DATE_WILD_CARD_KEY
+from ted_sws.event_manager.adapters.event_log_decorator import event_log
+from ted_sws.event_manager.model.event_message import TechnicalEventMessage
+
+DAG_KEY = f"fetch_notices_for_date_range_{datetime.now().isoformat()}"
```
Comment (CaptainOfHacks) [inline dags/fetch_notices_for_date_range.py:13]: This line executed once, I can't guarantee that this line will behave like DAG_RUN_ID (which is generated by airflow)  and is different at each DAG run.

File: dags/index_and_normalise_notice_worker.py [hunk 2]
```diff
@@ -11,7 +11,14 @@
...
 
 from dags import DEFAULT_DAG_ARGUMENTS
+from ted_sws.event_manager.adapters.event_log_decorator import event_log
+from ted_sws.event_manager.adapters.event_logger import EventLogger
+from ted_sws.event_manager.model.event_message import NoticeEventMessage
+from datetime import datetime
+from ted_sws.event_manager.services.logger_from_context import get_logger_from_dag_context, \
+    get_dag_args_from_context
 
+DAG_KEY = f"index_and_normalise_notice_worker_{datetime.now().isoformat()}"
```
Comment (CaptainOfHacks) [inline dags/index_and_normalise_notice_worker.py:21]: This line executed once, I can't guarantee that this line will behave like DAG_RUN_ID (which is generated by airflow) and is different at each DAG run.

File: ted_sws/event_manager/adapters/event_handler_config.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+from ted_sws import config
+from ted_sws.event_manager.adapters.event_handler import EventHandler, EventWriterToConsoleHandler, \
+    EventWriterToFileHandler, EventWriterToNullHandler, EventWriterToMongoDBHandler
+from ted_sws.event_manager.adapters.log import ConfigHandlerType
+
+NOW = datetime.now(timezone.utc)
+DEFAULT_LOGGER_NAME = "ROOT"
+DEFAULT_NULL_LOGGER_NAME = "NULL"
+DEFAULT_LOGGER_LOG_FILENAME = "./event_logs/{today}.log"
+# DEFAULT_LOGGER_LOG_FILENAME = "./event_logs/{today}/{hour}/{minute}.log" # EXAMPLE
```
Comment (CaptainOfHacks) [inline ted_sws/event_manager/adapters/event_handler_config.py:17]: remove comments

File: dags/index_and_normalise_notice_worker.py (hunk 2 above)
Comment (kaleanych) [reply dags/index_and_normalise_notice_worker.py:21]: this line is meant to keep connection between dag tasks and decorator logging

File: tests/unit/mapping_suite_processor/cli/test_cmd_mapping_suite_processor.py [hunk 4]
```diff
@@ -15,6 +15,7 @@ def test_mapping_suite_processor(cli_runner, fake_mapping_suite_id, file_system_
         fake_mapping_suite_id,
         "--opt-mappings-folder", file_system_repository_path
     ])
+    print("K :: ", response.output)
```
Comment (CaptainOfHacks) [inline tests/unit/mapping_suite_processor/cli/test_cmd_mapping_suite_processor.py:18]: I think we can remove this print

File: dags/index_and_normalise_notice_worker.py (hunk 2 above)
Comment (kaleanych) [reply dags/index_and_normalise_notice_worker.py:21]: the dag run_id is added by get_dag_args_from_context(context) service

File: ted_sws/event_manager/adapters/event_handler_config.py (hunk 3 above)
Comment (kaleanych) [reply ted_sws/event_manager/adapters/event_handler_config.py:17]: I've left it to show an example of log file naming

**Pull request merged**